tests/dummy/app/styles/vendor/_html5bp.scss

```diff
@@ -0,0 +1,245 @@
+/*! HTML5 Boilerplate v5.2.0 | MIT License | https://html5boilerplate.com/ */
```

Comment:
File added line-for-line from [compass-boilerplate](https://github.com/sir-dunxalot/compass-boilerplate/blob/master/lib/compass-boilerplate-vendor/_html5bp.scss).